src/index.js

```diff
@@ -46,7 +47,10 @@ const ApolloClientPlugin = createPlugin({
 
       const connectionLink =
         schema && __NODE__
-          ? new SchemaLink({schema})
+          ? new SchemaLink({
+            schema,
+            apolloContext: typeof context === 'function' ? context(ctx) : context,
```

Comment:
Sorry, I think this should still use the `context` key, and the other places should be renamed to `apolloContext`.